mocha-multi.js

```diff
@@ -35,8 +35,18 @@ process.exit = function(code) {
   process.on('exit', quit);
 }
 
-function MochaMulti(runner) {
-  var streams = initReportersAndStreams(runner, parseSetup());
+function MochaMulti(runner, options) {
+  var setup=[];
+  if( options && options.reporterOptions ) {
+    debug("options %j", options.reporterOptions);
+    var reporters=Object.keys(options.reporterOptions);
+    reporters.forEach(function(reporter) {
+      debug("%j %j", options.reporterOptions, reporter);
+      setup.push([ reporter, options.reporterOptions[reporter].stdout, options.reporterOptions[reporter].options ]);
+    });
+  }
+  debug("setup %j", setup);
+  var streams = initReportersAndStreams(runner, setup || parseSetup());
```

Comment:
Ah, the problem is here - `setup` is never falsy, so it will never use the environment variable.

Perhaps do something like:
```js
var setup;
if (options && options.reporterOptions) {
  setup = // ...
} else {
  setup = parseSetup()
}
```